tests/transpilation/and.opcode.gold.cairo

```diff
@@ -8,26 +9,28 @@ from starkware.cairo.common.serialize import serialize_word
 from starkware.cairo.common.uint256 import Uint256, uint256_and, uint256_eq
 
 func segment0{range_check_ptr, msize, memory_dict : DictAccess*}(stack : StackItem*) -> (
-        stack : StackItem*, evm_pc : Uint256):
+        stack : StackItem*, evm_pc : Uint256, output : Output):
     alloc_locals
     let stack0 = stack
     let (local __fp__, _) = get_fp_and_pc()
     let (local tmp0 : Uint256) = uint256_and(Uint256(63, 0), Uint256(46, 0))
     local newitem0 : StackItem = StackItem(value=tmp0, next=stack0)
-    return (stack=&newitem0, evm_pc=Uint256(-1, 0))
+    return (stack=&newitem0, evm_pc=Uint256(0, 0), output=Output(1, cast(0, felt*), 0))
 end
 
 func run_from{range_check_ptr, msize, memory_dict : DictAccess*}(
-        evm_pc : Uint256, stack : StackItem*) -> (res : Uint256):
+        evm_pc : Uint256, stack : StackItem*) -> (stack : StackItem*, output : Output):
     let (immediate) = uint256_eq(evm_pc, Uint256(0, 0))
     if immediate == 1:
-        let (stack, evm_pc) = segment0(stack=stack)
+        let (stack, evm_pc, output) = segment0(stack=stack)
+        if output.active == 1:
+            return (stack, output)
+        end
         return run_from(evm_pc=evm_pc, stack=stack)
     end
     let (immediate) = uint256_eq(evm_pc, Uint256(-1, 0))
     if immediate == 1:
-        let stack0 : StackItem* = stack
-        return (res=stack0.value)
+        return (stack, Output(1, cast(0, felt*), 0))
```

Comment:
I do understand what is happening in here, that it provides a null array for input, still `cast (0, felt*)`. Still have some kind of mental friction to read it as a `null`.

Reply:
I think I have none 😞  This is probably language related comment.